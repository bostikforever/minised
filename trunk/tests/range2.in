line A
line B
line C
and so on
