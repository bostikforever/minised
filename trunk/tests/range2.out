line A
and so on
